server/service/src/stocktake_line/insert.rs

```diff
@@ -112,6 +117,20 @@ fn check_stock_line_exists(
         .pop())
 }
 
+pub fn stocktake_difference(
```

Comment:
for me, the name of the fn doesn't fully describe it - a difference doesn't describe the change, just that something is different. more helpful to have `stocktake_reduction_amount` - that way it is clearer what a positive or negative result would mean. I was unsure when using the fn whether it was referring to stock - counted or the other way around.